recipe/meta.yaml

```diff
@@ -8,16 +8,18 @@ package:
 source:
   url: https://pypi.io/packages/source/s/slycot/slycot-{{ version }}.tar.gz
   sha256: {{ sha256 }}
+  patches:
+    - setupcfgin.patch
 
 build:
-  number: 8
+  number: 0
 
 requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
-    - scikit-build                           # [build_platform != target_platform]
+    - numpy !=1.23.0                        # [build_platform != target_platform]
```

Comment:
```suggestion
    - numpy 1.19                            # [build_platform != target_platform]
```

Reply:
I tried these locally, and that doesn't work for Python 3.10.  What I've just pushed succeed locally with Python 3.7 and 3.10, non-Docker build.  I can't get the docker builds to work; the build times out on a download (I think package repo information).

Is that conda_build_config.yaml used in these CI builds?  It *should* sort out Python 3.9, which is failing in CI.  That file also doesn't have an entry for Python 3.10 -- is Python 3.10 support not in yet?